components/cf-app-ssh/backend/glide.yaml

```diff
@@ -0,0 +1,11 @@
+package: github.com/SUSE/stratos-ui/components/cf-app-ssh/backend
+import:
+- package: github.com/Sirupsen/logrus
+  version: ^0.11.5
+- package: github.com/labstack/echo
+  version: ^2.1.0
+- package: github.com/satori/go.uuid
+  version: ^1.1.0
+- package: golang.org/x/crypto
+  subpackages:
+  - ssh
```

Comment:
Gorilla websocket packages are mentioned in the `glide.lock` file, but are absent from the `glide.yaml`. They should be present here as well.